### PR TITLE
Try to get sortition pool before creating a new one

### PR DESCRIPTION
**Pull request by lukasz-zimnoch**
Sortition pool existence should be checked before trying to create a new one during `lcl-initialize.js` script execution. This is a must when executing `local-setup` scripts because keep-network/tbtc#715 introduced sortition pool creation during `tbtc` contract migration. If we always create a sortition pool in that case, we can expect the `lcl-initialize.js` to fail.

### CODE REVIEW

File: solidity/scripts/lcl-initialize.js [hunk 1]
```diff
@@ -76,17 +76,32 @@ module.exports = async function () {
...
-      console.log(`created sortition pool for application: [${application}]`)
+      console.log(`getting sortition pool for application [${application}]`)
 
       sortitionPoolAddress = await bondedECDSAKeepFactory.getSortitionPool(
         application
       )
     } catch (err) {
-      console.error("failed to create sortition pool", err)
-      process.exit(1)
+      console.error("failed to get sortition pool", err, "; creating a new one")
```
Comment (nkuba) [inline solidity/scripts/lcl-initialize.js:85]: Should we do `process.exit(1)` in the next line?

Comment (lukasz-zimnoch) [reply solidity/scripts/lcl-initialize.js:85]: But we don't :)

Comment (nkuba) [reply solidity/scripts/lcl-initialize.js:85]: So shouldn't we?

Comment (lukasz-zimnoch) [reply solidity/scripts/lcl-initialize.js:85]: Nope, if we catch an error from `getSortitionPool` we should try to create a new one instead of exit.

Comment (nkuba) [reply solidity/scripts/lcl-initialize.js:85]: 🤦 Please forget

**Pull request merged**